de-lib/src/main/java/org/iplantc/de/theme/base/client/tools/NewToolRequestFormViewDefaultAppearance.java

```diff
@@ -50,4 +64,54 @@ public SafeHtml buildRequiredFieldLabel(final String label) {
         return SafeHtmlUtils.fromTrustedString(
                 "<span style='color:red; top:-5px;' >*</span> " + label); //$NON-NLS-1$
     }
+
+    @Override
+    public SafeHtml sameFileError(String filename) {
+        return toolMessages.sameFileError(filename);
+    }
+
+    @Override
+    public SafeHtml alert() {
+        return toolMessages.alert();
+    }
+
+    @Override
+    public SafeHtml fileSizeViolation(String filename) {
+        return toolMessages.fileSizeViolation(filename);
+    }
+
+    @Override
+    public SafeHtml maxFileSizeExceed() {
+        return toolMessages.maxFileSizeExceed();
+    }
+
+    @Override
+    public SafeHtml fileExistTitle() {
+        return toolMessages.fileExistTitle();
+    }
+
+    @Override
+    public SafeHtml fileExists(String dupeFiles) {
+        return toolMessages.fileExists(dupeFiles);
+    }
+
+    @Override
+    public String invalidFileName() {
+        return toolMessages.invalidFileName();
+    }
```

Comment:
A few of the above strings are duplicated in `DiskResourceMessages`.  Instead of duplicating them in `ToolMessages` could we include `DiskResourceMessages` in this appearance and reuse those strings?

Reply:
👍